addons/delivery/views/delivery_view.xml

```diff
@@ -207,16 +132,13 @@
                 </form>
             </field>
         </record>
-        <record id="view_delivery_grid_line_tree" model="ir.ui.view">
-            <field name="name">delivery.grid.line.tree</field>
-            <field name="model">delivery.grid.line</field>
+        <record id="view_delivery_price_rule_tree" model="ir.ui.view">
+            <field name="name">delivery.price.rule.tree</field>
+            <field name="model">delivery.price.rule</field>
             <field name="arch" type="xml">
-                <tree string="Grid Lines">
+                <tree string="Price Rules">
                     <field name="sequence" widget="handle" />
                     <field name="name"/>
-                    <field name="type"/>
-                    <field name="operator"/>
-                    <field name="max_value" nolabel="1"/>
                     <field name="list_price"/>
                     <field name="standard_price"/>
```

Comment:
it this is a cost price remove this field from view and code, i don;t think this value used anywhere in computation.